lib/qxcli/commands/Create.js

```diff
@@ -232,7 +233,7 @@ qx.Class.define("qxcli.commands.Create", {
         fs.mkdirSync(appdir);
       }
       // make qxpath relative
-      values.qxpath = path.relative(appdir, values.qxpath);
+      values.qxpath = path.relative(appdir, values.qxpath).replace(new RegExp('\\' + path.sep, 'g'), '/');
```

Comment:
This feels a bit hackish. Maybe we should use https://www.npmjs.com/package/upath instead of `path`? @johnspackman

Reply:
Then we should change the other places too.

Reply:
If @johnspackman agrees, we can simply switch from `path` to  `upath` which will take care of it anywhere and make `replace()` like this unnecessary.

Reply:
then we should change to upath everywhere, should'nt we? In that case i would suggest another pull request.

Reply:
Yes, let's do this in another PR after testing that everything keeps working with the drop-in-replacement. But I'd like to wait for @johnspackman to agree. In the meantime, would it be ok to remove the replace() ? Or would that break something for you?

Reply:
yes, compile of new created Projects with windows will not work because of the wrong qooxdoo path. But i can revert and fix it later. 
I only want help to make the cli production ready, no need for creating new Project in the Moment.